src/main/java/dk/kb/ginnungagap/cumulus/CumulusPreservationUtils.java

```diff
@@ -90,10 +98,12 @@ public static void initIntellectualEntityUUID(CumulusRecord record) {
         String uuid = record.getFieldValueOrNull(
                 Constants.FieldNames.RELATED_OBJECT_IDENTIFIER_VALUE_INTELLECTUEL_ENTITY);
         if(uuid == null || uuid.isEmpty()) {
-            record.setStringValueInField(Constants.FieldNames.RELATED_OBJECT_IDENTIFIER_VALUE_INTELLECTUEL_ENTITY, 
-                    UUID.randomUUID().toString());
+            uuid = UUID.randomUUID().toString();
+            log.trace("Setting uuid value for 'relatedObjectIdentifierValue_intellectualEntity': " + uuid);
+            record.setStringValueInField(Constants.FieldNames.RELATED_OBJECT_IDENTIFIER_VALUE_INTELLECTUEL_ENTITY,
+                    uuid);
         } else {
-            log.trace("Already has a value for the related intellectual object of the representation");
+            log.trace("Already has a value for the 'relatedObjectIdentifierValue_intellectualEntity': " + uuid);
```

Comment:
The string constant RELATED_OBJECT_IDENTIFIER_VALUE_INTELLECTUEL_ENTITY could also have been used here.